resource/history/mage-os/magento2-base/1.0.6.json

```diff
@@ -0,0 +1,572 @@
+{
+    "name": "magento/magento2-base",
+    "description": "Magento 2 Base (Community Edition)",
+    "type": "magento2-component",
+    "license": [
+        "OSL-3.0",
+        "AFL-3.0"
+    ],
+    "version": "2.4.7-p4",
```

Comment:
Comparing to https://github.com/mage-os/generate-mirror-repo-js/pull/183/files for 1.0.5.json, I think we need to wait until we build and publish the actual 1.0.6 release so we can pull the `magento/magento2-base` JSON from that, including its pinned versions. This current one is quite different from 1.0.5's, so I don't think it's right.

Reply:
ok so this whole PR should be made AFTER a release, unlike the mirror, right?

Reply:
Exactly correct.

Reply:
good, I'll update the guide accordinly